airbyte-integrations/bases/base-normalization/normalization/transform_catalog/stream_processor.py

```diff
@@ -322,7 +323,7 @@ def generate_json_parsing_model(self, from_table: str, column_names: Dict[str, T
     {{ field }},
   {%- endfor %}
     _airbyte_emitted_at
-from {{ from_table }}
+from {{ from_table }} as {{ table_alias }}
```

Comment:
YOu need to introduce `table_alias` in only the extract_json cte not all of them